application/client/src/app/service/session/dependencies/search/state.ts

```diff
@@ -1,26 +1,40 @@
-import { FilterRequest } from './filters/request';
 import { IFilter } from '@platform/types/filter';
 import { Subjects, Subject } from '@platform/env/subscription';
 import { Search } from '@service/session/dependencies/search';
 
 import * as obj from '@platform/env/obj';
 
+export interface IFinish {
+    found: number;
+    error?: string;
+}
+
+export interface ISearchInfo {
+    state: ESearchState;
+    found?: number;
+}
```

Comment:
Doesn't make sense to store in the state `found`, because the amount of found matches will be changes after the initial search. I think actual `found` you always can take from `map` (`session.search.map`), but check it please